tests/fixtures/plugin/build.atpkg

```diff
@@ -0,0 +1,29 @@
+;; Copyright (c) 2016 Anarchy Tools Contributors.
+;;
+;; Licensed under the Apache License, Version 2.0 (the "License");
+;; you may not use this file except in compliance with the License.
+;; You may obtain a copy of the License at
+;;
+;;   http:;;www.apache.org/licenses/LICENSE-2.0
+;;
+;; Unless required by applicable law or agreed to in writing, software
+;; distributed under the License is distributed on an "AS IS" BASIS,
+;; WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+;; See the License for the specific language governing permissions and
+;; limitations under the License.
+
+(package
+  :name "plugin"
+  
+  :tasks {
+    :a {:tool "nop" }
+    :default {
+      :tool "echo.plugin"
```

Comment:
It's not that they are more important, it's that explicit is better than implicit.  If you have a build.atpkg from unknown provenance (e.g. your dependency, etc.) seeing `.plugin` makes clear where the tool lives, so you know to install it.  Further, we can give a good error, further atpm might make use of this information to suggest installing the tool in some way.  Whereas if there is no explicit difference between builtin and external tools there is no way to distinguish whether this is a builtin tool unknown to an older atbuild (in which case the resolution is to upgrade) or a plugin (in which case the resolution is to install it).

Secondarily, we have a namespacing issue.  `packageframework` in particular is a tool I feel should probably live in a plugin, but it may be appropriate to deprecate the builtin version slowly which means they need to have distinct names.  Namespacing isn't the only way to achieve this, but it seems like an okay one.